cmd/soroban-rpc/main.go

```diff
@@ -134,6 +134,13 @@ func main() {
 			},
 			Usage: "comma-separated list of stellar history archives to connect with",
 		},
+		{
+			Name:      "friendbot-url",
+			Usage:     "URL where friendbot requests should be sent",
```

Comment:
The description here isn't incorrect, but I'm not sure that it's what we want in this context.
i.e. I was thinking something in the lines of "The friendbot URL which would be returned by getNetwork endpoint".

The reason for that is that the soroban-rpc doesn't use this field in any way other then copy-paste it into it's responses.

Reply:
I thought it will be used by soroban-rpc for doing account funding for test networks. If it's not going to use it itself in anyway, it seems pretty strange for the soroban-rpc to offer it up as data or know about it at all.

Reply:
The idea is that the client can call `getNetwork` to determine if there is a friendbot available on this network, and figure out how to request from it. This is replacing the proposed soroban-rpc `requestAirdrop` method, which would do the same thing server-side.

Reply:
👍